micro-ui/web/micro-ui-internals/packages/modules/core/src/pages/employee/QuickStart/QuickSetup.js

```
@@ -0,0 +1,91 @@
+    </div>
+  );
+};
+
+const FAQ = ({ key, title, content, faqs }) => {
+  return (
+    <div>
+      <CardSectionHeader>{title}</CardSectionHeader>
+      <CardSectionSubText>{content}</CardSectionSubText>
+      <div style={{ width: "100%" }}>
+        {faqs.map((faq, i) => (
+          <FaqComponent key={"faq_" + i} question={faq.question} answer={faq.answer} lastIndex={i === faqs?.length - 1} />
```

Biome

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)